### ISSUE TITLE
Add http response status code to org.eclipse.jetty.server.RequestLog

### ISSUE THREAD

**Issue by themarcelor** [apache/druid#12112]
Can you please augment the jetty logger to also present the HTTP response code? 🙏🏼 

We just need to add `response.getStatus()` to this class:

```public class JettyRequestLog extends AbstractLifeCycle implements RequestLog
{
  private static final Logger logger = new Logger("org.apache.druid.jetty.RequestLog");

  @Override
  public void log(Request request, Response response)
  {
    if (logger.isDebugEnabled()) {
      logger.debug(
          "%s %s %s %s %s",
          request.getRemoteAddr(),
          request.getMethod(),
          request.getHttpURI().toString(),
          request.getProtocol(),
          response.getStatus()
      );
    }
  }
}
```
This would be extremely helpful to calculate the SR (Success Rate) based on the percentage of non-http-5xx requests.

**Comment by FrankChen021** [comment]
You can open a PR to do it by yourself, I think it will be accepted since it makes sense.

**Comment by themarcelor** [comment]
There you go:
https://github.com/apache/druid/pull/12116

Please provide some handholding around testing.

**Comment by FrankChen021** [comment]
Closed via #12116
